docs/run.md

```
@@ -35,16 +35,48 @@ Run the following commands to configure the minimum required firewall rules,
+See [Create DEB Packages](../docs/build.md#create-deb-packages) for more information.
+
+```bash
+DEB_REPO_DIR=/tmp/microshift-rpms/deb
+sudo ./src/deb/install.sh "${DEB_REPO_DIR}"
+```
+
+The following optional DEB packages are available in the repository.
+
+| Package            | Description                | Comments |
+|--------------------|----------------------------|----------|
+| microshift-topolvm | TopoLVM CSI                |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)